Adding logic for existing columns

gcyoung1 opened the pull request.

kipohl commented:
@gcyoung1 can you run it on an example to make sure it works - also why would this ever happen 
f"ERROR: Overwriting column {col_name} with new values.

gcyoung1 commented:
@kipohl Here's a test run in production:
```
ncanda@pipeline-back[pipeline_back_1]:/sibis-software/python-packages/sibispy/cmds (bug/shared_variables_between_csvs)$ /sibis-software/ncanda-data-integration/scripts/redcap/export_measures --subject D-00312-M-9 -v --progress-bar /fs/ncanda-share/cases
INFO: Modified link for A-00082-F-3 from NCANDA_S01168 to NCANDA_S00617 !
INFO: Modified link for A-00129-M-1 from NCANDA_S01063 to NCANDA_S00776 !
INFO: Ignoring link B-00359-F-1 to NCANDA_S01104 as already already linked to NCANDA_S00555 !
INFO: Ignoring link B-00368-M-9 to NCANDA_S01107 as already already linked to NCANDA_S00480 !
INFO: Modified link for B-00381-F-2 from NCANDA_S01091 to NCANDA_S00558 !
INFO: Ignoring link B-80556-F-6 to NCANDA_S01024 as already already linked to NCANDA_S01001 !
INFO: Modified link for C-70087-M-8 from NCANDA_S01166 to NCANDA_S00548 !
INFO: Ignoring link C-70158-F-4 to NCANDA_S01164 as already already linked to NCANDA_S00693 !
INFO: Ignoring link C-70165-F-9 to NCANDA_S01058 as already already linked to NCANDA_S00747 !
INFO: Ignoring link D-00094-M-5 to NCANDA_S01011 as already already linked to NCANDA_S00270 !
INFO: Modified link for D-00133-F-0 from NCANDA_S01099 to NCANDA_S00425 !
INFO: Modified link for D-00141-M-9 from NCANDA_S01147 to NCANDA_S00292 !
INFO: Ignoring link D-00159-F-2 to NCANDA_S01100 as already already linked to NCANDA_S00327 !
INFO: Modified link for E-00018-F-2 from NCANDA_S01165 to NCANDA_S00033 !
INFO: Modified link for E-00051-M-9 from NCANDA_S01031 to NCANDA_S00060 !
INFO: Ignoring link E-00084-F-7 to NCANDA_S01035 as already already linked to NCANDA_S00142 !
INFO: Modified link for E-00090-F-1 from NCANDA_S01057 to NCANDA_S00129 !
INFO: Modified link for E-00186-M-0 from NCANDA_S01094 to NCANDA_S00193 !
INFO: Modified link for E-00348-F-2 from NCANDA_S01105 to NCANDA_S00446 !
INFO: Ignoring link E-00445-M-9 to NCANDA_S01167 as already already linked to NCANDA_S00442 !
INFO: Ignoring link E-00649-M-2 to NCANDA_S01078 as already already linked to NCANDA_S00818 !
INFO: Ignoring link E-00879-M-8 to NCANDA_S01076 as already already linked to NCANDA_S00672 !
INFO: Modified link for E-00880-M-1 from NCANDA_S01101 to NCANDA_S00662 !
INFO: Modified link for X-99997-F-9 from NCANDA_S00240 to NCANDA_S00157 !
INFO: Ignoring link X-99997-F-9 to NCANDA_S00212 as already already linked to NCANDA_S00157 !
INFO: Ignoring link X-99997-F-9 to NCANDA_S00201 as already already linked to NCANDA_S00157 !
INFO: Ignoring link X-99997-F-9 to NCANDA_S01018 as already already linked to NCANDA_S00157 !
INFO: Modified link for X-99998-M-9 from NCANDA_S00208 to NCANDA_S00200 !
INFO: Ignoring link X-99998-M-9 to NCANDA_S00765 as already already linked to NCANDA_S00200 !
INFO: Modified link for X-99998-M-9 from NCANDA_S00200 to NCANDA_S00147 !
INFO: Ignoring link X-99998-M-9 to NCANDA_S00158 as already already linked to NCANDA_S00147 !
INFO: Ignoring link X-99999-F-9 to NCANDA_S00040 as already already linked to NCANDA_S00030 !
INFO: Ignoring link X-99999-F-9 to NCANDA_S00053 as already already linked to NCANDA_S00030 !
INFO: Modified link for X-99999-F-9 from NCANDA_S00030 to NCANDA_S00023 !
INFO: Ignoring link X-99999-F-9 to NCANDA_S00035 as already already linked to NCANDA_S00023 !
Exporting 16 REDCap records.
  0%|                                         | 0/16 [00:00<?, ?subjects/s]Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport1_demographics.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/parentreport_pei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport2_ssq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport2_leaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport2_pds.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport2_sog.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport1_npchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport2_brief.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport1_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/parentreport_aans.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport2_pgd.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/parentreport_cbcl.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport1_cddr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport2_dhr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport1_tbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport2_pss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/baseline/measures/youthreport2_ptm.csv
 12%|████▏                            | 2/16 [00:04<00:27,  1.97s/subjects]Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_ctq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_ula.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_ypa.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/parentreport_pdi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1b_asr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_aay.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_shq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1_fhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_prr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1b_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_psqi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_acc.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1_ciq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1_ses.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_tipi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_aswhs.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_rpir.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_pm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_aeq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1b_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_chks.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_maaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_dtcq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_upps.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_scsm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/parentreport_sesp.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/parentreport_ptbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_rsq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_pmi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1b_ysr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_pwmkcr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_hss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/parentreport_pfhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1_sre.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_casq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_aces.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_sbq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_aeas.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1_demographics.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/parentreport_pei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_ssq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_leaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_pds.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_sog.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1_npchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_brief.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/parentreport_aans.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_pgd.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/parentreport_cbcl.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1_cddr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_dhr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport1_tbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_pss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_1y/measures/youthreport2_ptm.csv
 25%|████████▎                        | 4/16 [00:10<00:26,  2.24s/subjects]Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_ctq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_ula.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_ypa.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/parentreport_pdi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1b_asr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_aay.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_shq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1_fhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_prr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1b_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_psqi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_acc.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1_ciq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1_ses.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_tipi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_aswhs.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_rpir.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_pm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_aeq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1b_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_chks.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_maaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_dtcq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_upps.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_scsm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/parentreport_sesp.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/parentreport_ptbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_rsq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_pmi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1b_ysr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_pwmkcr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_hss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/parentreport_pfhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1_sre.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_casq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_aces.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_sbq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_aeas.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1_demographics.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/parentreport_pei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_ssq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_leaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_pds.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_sog.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1_npchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_brief.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/parentreport_aans.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_pgd.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/parentreport_cbcl.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1_cddr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_dhr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport1_tbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_pss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_2y/measures/youthreport2_ptm.csv
 38%|████████████▍                    | 6/16 [00:15<00:22,  2.27s/subjects]Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_ctq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_ula.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_ypa.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/parentreport_pdi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1b_asr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_aay.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_shq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1_fhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_prr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1b_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_psqi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_acc.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1_ciq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1_ses.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_tipi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_aswhs.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_rpir.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_pm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_aeq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1b_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_chks.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_maaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_dtcq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_upps.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_scsm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/parentreport_sesp.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/parentreport_ptbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_rsq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_pmi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1b_ysr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_pwmkcr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_hss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/parentreport_pfhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1_sre.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_casq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_aces.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_sbq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_aeas.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1_demographics.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/parentreport_pei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_ssq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_leaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_pds.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_sog.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1_npchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_brief.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/parentreport_aans.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_pgd.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/parentreport_cbcl.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1_cddr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_dhr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport1_tbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_pss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_3y/measures/youthreport2_ptm.csv
 50%|████████████████▌                | 8/16 [00:21<00:18,  2.32s/subjects]Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_ctq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_ula.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_ypa.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/parentreport_pdi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1b_asr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_aay.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_shq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1_fhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_prr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1b_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_psqi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_acc.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1_ciq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1_ses.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_tipi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_aswhs.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_rpir.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_pm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_aeq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1b_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_chks.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_maaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_dtcq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_upps.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_scsm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/parentreport_sesp.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/parentreport_ptbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_rsq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_pmi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1b_ysr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_pwmkcr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_hss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/parentreport_pfhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1_sre.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_casq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_aces.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_sbq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_aeas.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1_demographics.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/parentreport_pei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_ssq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_leaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_pds.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_sog.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1_npchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_brief.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/parentreport_aans.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_pgd.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/parentreport_cbcl.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1_cddr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_dhr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport1_tbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_pss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_4y/measures/youthreport2_ptm.csv
 62%|████████████████████            | 10/16 [00:26<00:14,  2.38s/subjects]Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_ctq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_ula.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_ypa.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/parentreport_pdi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1b_asr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_aay.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_shq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1_fhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_prr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1b_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_psqi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_acc.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1_ciq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1_ses.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_tipi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_aswhs.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_rpir.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_pm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_aeq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1b_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_chks.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_maaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_dtcq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_upps.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_scsm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/parentreport_sesp.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/parentreport_ptbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_rsq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_pmi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1b_ysr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_pwmkcr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_hss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/parentreport_pfhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1_sre.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_casq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_aces.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_sbq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_aeas.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1_demographics.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/parentreport_pei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_ssq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_leaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_pds.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_sog.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1_npchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_brief.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/parentreport_aans.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_pgd.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/parentreport_cbcl.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1_cddr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_dhr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport1_tbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_pss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_5y/measures/youthreport2_ptm.csv
 75%|████████████████████████        | 12/16 [00:32<00:09,  2.31s/subjects]Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_ctq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_ula.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_ypa.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/parentreport_pdi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1b_asr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_aay.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_shq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1_fhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_prr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1b_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_psqi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_acc.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1_ciq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1_ses.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_tipi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_aswhs.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_rpir.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_pm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_aeq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1b_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_chks.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_maaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_dtcq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_upps.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_scsm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/parentreport_sesp.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/parentreport_ptbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_rsq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_pmi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1b_ysr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_pwmkcr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_hss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/parentreport_pfhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1_sre.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_casq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_aces.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_sbq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_aeas.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1_demographics.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/parentreport_pei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_ssq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_leaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_pds.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_sog.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1_npchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_brief.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/parentreport_aans.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_pgd.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/parentreport_cbcl.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1_cddr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_dhr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport1_tbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_pss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_6y/measures/youthreport2_ptm.csv
 88%|████████████████████████████    | 14/16 [00:37<00:04,  2.30s/subjects]Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_ctq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_ula.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_ypa.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/parentreport_pdi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1b_asr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_aay.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_shq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1_fhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_prr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1b_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_psqi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_acc.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1_ciq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1_ses.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_tipi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_aswhs.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_rpir.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_pm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_aeq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/clinical.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1b_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_chks.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_yei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_maaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_dtcq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_upps.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_scsm.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/parentreport_sesp.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/parentreport_ptbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_rsq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_pmi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1b_ysr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_pwmkcr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_hss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/parentreport_pfhi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1_sre.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_casq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_aces.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_sbq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_aeas.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1_demographics.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/parentreport_pei.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_ssq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_leaq.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_pds.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_sog.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1_npchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_brief.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1_bdchecklist.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/parentreport_aans.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_pgd.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/parentreport_cbcl.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1_cddr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_dhr.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport1_tbi.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_pss.csv
Updated /fs/ncanda-share/cases/NCANDA_S00679/standard/followup_7y/measures/youthreport2_ptm.csv
100%|████████████████████████████████| 16/16 [00:43<00:00,  2.74s/subjects]
```

I don't think that error will ever arise, but I haven't traced the code back thoroughly enough to be sure. I can take out the slog call if you're sure it won't.

kipohl commented:
@gcyoung1 I do not think your fix is correct 
a) /sibis-software/ncanda-data-integration/scripts/redcap/export_measures --subject D-00312-M-9 --export youthreport2_aeas,youthreport2 -v /fs/ncanda-share/cases
runs fine 
b) my guess is that by mistake one of the txt files included a variable twice - in that case the txt file needs fixing

kipohl commented:
so requires proper error message

gcyoung1 commented:
@kipohl yup, you were right. youthreport1_demographics.txt had year twice:
```
ncanda@pipeline-back[pipeline_back_1]:/fs/ncanda-share/ncanda-operations/redcap_to_casesdir (master)$ grep 'youthreport1_year' ./youthreport1* 
./youthreport1.txt:youthreport1_year
./youthreport1_bdchecklist.txt:youthreport1_year
./youthreport1_cddr.txt:youthreport1_year
./youthreport1_ciq.txt:youthreport1_year
./youthreport1_demographics.txt:youthreport1_year
./youthreport1_demographics.txt:youthreport1_year
./youthreport1_fhi.txt:youthreport1_year
./youthreport1_npchecklist.txt:youthreport1_year
./youthreport1_ses.txt:youthreport1_year
./youthreport1_sre.txt:youthreport1_year
./youthreport1_tbi.txt:youthreport1_year
```
We can delete this PR then. Do you want me to add a more descriptive error message for if this comes up again? @kipohl

kipohl commented:
sounds good @gcyoung1 once you create the PR please delete this one
